core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -223,6 +239,39 @@ oppia.controller('FeedbackTab', [
               });
             };
 
+            $scope.editSuggestion = function() {
+              $scope.suggestionEditorIsShown = true;
+              $scope.isNotHandled = true;
+              $scope.saveButtonIsShown = true;
+            };
+
+            $scope.saveSuggestion = function() {
+              $scope.suggestionEditorIsShown = false;
+              $scope.saveButton = false;
+
+              url = UrlInterpolationService.interpolateUrl(
+                '/suggestionactionhandler/edit/<target_id>/<suggestion_id>', {
+                  target_id: $scope.target_id,
+                  suggestion_id: $scope.suggestion_id,
+                });
+
+              data = {
+                action: ACTION_EDIT_SUGGESTION,
+                summary_message: $scope.summaryMessage,
+                change: {
+                  cmd: 'edit_state_property',
+                  property_name: 'content',
+                  state_name: $scope.stateName,
+                  old_value: $scope.currentContent,
+                  new_value: {
+                    html: $scope.newContent
+                  }
+                }
+              };
+
+              $http.put(url, data); // sending changed suggestion to server.
```

Comment:
Capitalize the first letter of the comment, maybe you can move this to line 271.

Reply:
ok sir